message_ix/model/MESSAGE/model_core.gms

```diff
@@ -1317,33 +1351,33 @@ SHARE_CONSTRAINT_MODE_LO(shares,node,tec,mode,year,time)$(
 SHARE_CONSTRAINT_COMMODITY_UP(shares,node_share,year,time)$( share_commodity_up(shares,node_share,year,time) )..
 * activity by type_tec_share technologies with map_shares_generic_share entries and a specific mode
     SUM( (node,location,type_tec_share,tec,vintage,mode,commodity,level,time2)$(
-	( map_shares_commodity_share(shares,node_share,node,type_tec_share,mode,commodity,level) OR
-	   map_shares_commodity_share(shares,node_share,node,type_tec_share,'all',commodity,level) ) AND
+        ( map_shares_commodity_share(shares,node_share,node,type_tec_share,mode,commodity,level) OR
+           map_shares_commodity_share(shares,node_share,node,type_tec_share,'all',commodity,level) ) AND
```

Comment:
Are these tab → space changes in whitespace (or vice versa)? What is our standard?

Reply:
No tabs! =)

Reply:
dont't seem to be tabs, just harmonising the indenting level for the extensive `if` declarations (but indenting with spaces)